ui-components/src/global/vendor/ag_grid.scss

```diff
@@ -110,15 +110,18 @@
   }
 }
 
-.data-pager {
+.data-pager-container {
   @apply p-4;
   @apply flex;
   @apply items-center;
+  @apply bg-gray-200;
+}
+
+.include-border {
```

Comment:
very optional thought:
we could nest this inside `.data-pager-container` like
```
.data-pager-container {
  &.include-border {
  ...
  }
}
```
so it generates css like
```
.data-pager-container { ... }
.data-pager-container.include-border {
   // only apply these rules to elements that have both the data-pager-container and include-border classes
}
```

that only real benefit is it makes it clear that the use case for `include-border` is in the context of `data-pager-container`